### ISSUE TITLE
No GUI menu action to quit app while running in background is activated

### ISSUE THREAD

**Issue by UnderEu** [GeopJr/Tuba#1071]
### Describe the bug

There is no option to close the app in the app menu. When there's an update or if I need to close the app entirely for whatever reason, I'm forced to open a terminal, grab the PID and kill it.
Clicking on the X icon on the window corner does not close the app, just hides it.

### Steps To Reproduce

Open the app menu
There is no "Close" option

### Logs and/or Screenshots

![Untitled](https://github.com/user-attachments/assets/a46f850b-46e9-4428-be58-c6efdcb5d120)

### Instance Backend

Mastodon

### Operating System

Ubuntu MATE 22.04

### Package

Flatpak

**Comment by GeopJr** [comment]
Thanks for raising this issue!

Do you happen to have this option enabled? If so, that's what causing it:

![image](https://github.com/user-attachments/assets/af4daef0-2cc0-4231-8af1-b12eea541bf8)

**Comment by UnderEu** [comment]
> [...]
> 
> Do you happen to have this option enabled? If so, that's what causing it:
> 
> ![image](https://private-user-images.githubusercontent.com/18014039/350179872-af4daef0-2cc0-4231-8af1-b12eea541bf8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjEzNDkwMDksIm5iZiI6MTcyMTM0ODcwOSwicGF0aCI6Ii8xODAxNDAzOS8zNTAxNzk4NzItYWY0ZGFlZjAtMmNjMC00MjMxLThhZjEtYjEyZWVhNTQxYmY4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzE5VDAwMjUwOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTlkNDM1ZWEyMDQzMzM2MjdiMmU4ODQ4OWMxZjE3OWQzOGY2MDZlNjExNmQ5ZDU4ZmUxMmFkNjU2OGI0MTk5MmMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.y8Q6aI7EL481HWPXuVXDuR5VMb-O3XB2bHGJU5tBu4M)

Yes, I have it and I want it to work for the notifications - explains why I have to kill the app manually via Terminal but, enabled or disabled, there's still no "Close" option in the menu.

**Comment by GeopJr** [comment]
I don't really see the need for it to be honest, but it's not set in stone, I can add it if needed.

When Tuba is running in the background using flatpak, it should trigger the background apps portal. That puts it, for example for GNOME, here:

![image](https://github.com/user-attachments/assets/0e091feb-d371-45e8-952f-df93eeb1fcd9)

and similarly for Plasma. From there you can kill it.

Now, you mentioned that you use MATE, considering its maintenance status, if they don't support that, I'm not going to accommodate for it much.

You can always <kbd>Ctrl</kbd><kbd>Q</kbd> to close the app.

**Comment by nekohayo** [comment]
For what it's worth, at least since it now allows running in the background, Showtime does show a menu item for the corresponding keyboard shortcut:

![Image](https://github.com/user-attachments/assets/81b8f28d-3fcf-41e9-a112-a5566fef0267)

…even if Showtime only stays in background if actively playing music (because they would not be consuming resources while paused anyway).

In Tuba's case, there is no "pause / stop refreshing stuff on the network" action, unlike a music player. In a certain way, I see Tuba closer to the usecase of a bittorrent client or an Epiphany web app than a music player. I could imagine why sometimes users would be looking for an explicit quit action in the app's GUI…

**Comment by GeopJr** [comment]
I still don't really see the point of it when the only time is needed is when
1. 'Run in the background' is enabled
2. Your desktop doesn't support the background portal

I'm not really interested on bikeshedding it further so I'll add it, however, if the menu items reach an overwhelming amount, it's the first one to go:

![Image](https://github.com/user-attachments/assets/929f068b-d8b7-4221-8ae7-eabbc34900cb)
